dev/VSIX/ProjectTemplates/Desktop/CppWinRT/PackagedApp/BlankApp/WinUI.Desktop.CppWinRT.BlankApp.vstemplate

```diff
@@ -2,8 +2,8 @@
 <VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
   <TemplateData>
     <Hidden>true</Hidden>
-    <Name>Blank App (WinUI 3 in Desktop)</Name>
-    <Description>A project for creating a Desktop app based on the Windows UI Library (WinUI 3).</Description>
+    <Name>WinUI Blank App</Name>
```

Comment:
Same here, Can we use resx file for Name, description in these templates as well ?

Reply:
Great catch. Fixed!